src/input.rs

```diff
@@ -137,33 +138,33 @@ pub fn update_joystick(
 
 pub fn update_joystick_by_mouse(
     mouse_button_input: Res<Input<MouseButton>>,
-    mut cursor_evr: EventReader<CursorMoved>,
+    mut mousebtn_evr: EventReader<MouseButtonInput>,
```

Comment:
This does not alter the current operation? is everything correct with the examples?

Reply:
I just tested it and it works great, thank you very much for the fix